source/dotnet/Library/AdaptiveCards/AdaptiveCardConverter.cs

```diff
@@ -10,6 +10,9 @@ public class AdaptiveCardConverter : JsonConverter, ILogWarnings
     {
         public List<AdaptiveWarning> Warnings { get; set; } = new List<AdaptiveWarning>();
 
+        // TODO: temporary warning code for fallback card. Remove when common set of error codes created and integrated.
+        private enum WarningStatusCode { UnsupportedSchemaVersion = 7 };
```

Comment:
I'm fine with this for this commit so we don't muddy up the rest of this PR. However, I think as your next change we should change `AdaptiveWarning` to take a `WarningStatusCode` and fix it up. :)